entity-management-tests/src/integration-test/java/eu/europeana/entitymanagement/web/EntityRetrievalIT.java

```diff
@@ -504,12 +509,14 @@ public void retrieveOrganizationJsonExternalShouldBeSuccessful() throws Exceptio
     mockMvc
         .perform(
             get(IntegrationTestUtils.BASE_SERVICE_URL + "/" + requestPath + ".jsonld")
-                .param(WebEntityConstants.QUERY_PARAM_PROFILE, "external")
+                .param(WebEntityConstants.QUERY_PARAM_PROFILE, "external, dereference")
```

Comment:
please ad a test without dereference profile and verify that the dereferencing is not performed